Week2/homework/gradeCal.js

```diff
@@ -0,0 +1,27 @@
+ 'use strict'
+ const prompt = require('prompt');
+ prompt.start();
+// I installed npm prompt module to take score from user..
+// I wrote a callback function for converting the score to grade
+ function getGrade(score) {
+    let grade;
+    if (0 <= score && score <= 49) {
+        grade = 'F';
+    } else if (score <= 59) {
+        grade = 'E';
+    } else if (score <= 69) {
+        grade = 'D';
+    } else if (score <= 79) {
+        grade = 'C';
+    } else if (score <= 89) {
+        grade = 'B';
+    } else if (score <= 100) {
+        grade = 'A';
+    } 
+    return grade;
+}
+ console.log("Enter your score :");
+ const score = prompt.get(['score'], function (err, result) {
+   const myGrade= getGrade(result.score); // calling getGrade function with score and assigning it to myGrade
+   console.log("You got a " + myGrade + " (" + result.score + "%)");
+ });
```

Comment:
👍 for logic but I don't think this would run successfully on the browser as `require` is not a valid object which the browser understands. May I ask where did you find about using `require('prompt')`?

Reply:
Yes I know it will not work on browser. "Use node from the command line to test if your code works as expected". This sentence is written in the Readme file. I just wanted to take score from user. The 'pop-up prompt' doesn't work without browser and I cannot test my code. I asked David for help in the class last week so we did that part together. I understand exactly what we did. Thank you for your attention and review.🙂